src/patternfly/components/Tile/tile.scss

```diff
@@ -65,32 +65,26 @@
   }
 
   // stylelint-disable
-  &[disabled] {
+  &.pf-m-disabled {
+    --pf-c-tile--BackgroundColor: var(--pf-c-tile--m-disabled--BackgroundColor);
+    --pf-c-tile__title--Color: var(--pf-c-tile--m-disabled__title--Color);
+    --pf-c-tile__body--Color: var(--pf-c-tile--m-disabled__body--Color);
+    --pf-c-tile--before--BorderWidth: 0;
+
     pointer-events: none;
     cursor: not-allowed;
 
-    --pf-c-tile--BackgroundColor: var(--pf-c-tile--disabled--BackgroundColor);
-    --pf-c-tile__title--Color: var(--pf-c-tile--disabled__title--Color);
-    --pf-c-tile__body--Color: var(--pf-c-tile--disabled__body--Color);
-
-    &::before {
-      border: 0;
-    }
-
     .pf-c-tile__header.pf-m-stacked .pf-c-tile__icon {
       --pf-c-tile__icon--Color: var(--pf-c-tile--disabled__header--m-stacked__icon--Color);
```

Comment:
does this need to be set on `.pf-c-tile__header.pf-m-stacked .pf-c-tile__icon` or could we just set it under `.pf-m-disabled` as `--pf-c-tile__icon--Color: var(--pf-c-tile--m-disabled__header__icon--Color);`